app/models/subnet.rb

```diff
@@ -121,7 +121,19 @@ class Jail < ::Safemode::Jail
   def self.network_reorder(order_string = 'network')
```

Comment:
Should this API expose `ASC`/`DESC` explicitly to simplify parsing?
```suggestion
  def self.network_reorder(column = :network, order = nil)
```

Reply:
We just allow the column to be `network` in the controller. I think we should change the signature to `order: nil` that does the actual SQL code and have another method parse the order param and call `network_reorder(order: :asc)` or `network_reorder(order: :desc)`. But to be honest: I believe we're overengineering this. It's not in the scope of this PR to refactor the whole method.

Reply:
Agreed.